front/app/components/ContentContainer/index.tsx

```diff
@@ -39,13 +39,14 @@ interface Props {
   className?: string;
   mode?: 'oldPage' | 'banner' | 'page' | 'text';
   maxWidth?: number | string;
+  testId?: string;
 }
 
 interface State {}
 
 export default class ContentContainer extends PureComponent<Props, State> {
   render() {
-    const { mode, id, className, children } = this.props;
+    const { mode, id, className, children, testId } = this.props;
```

Comment:
Passing data-testid as a prop feels a bit out of place - it rarely needs to be dynamic. Also - normally you would get and test what is inside the container - not the container itself?